nservicebus/upgrades/azure-functions-service-bus-in-process-isolated-worker.md

```diff
@@ -146,3 +146,26 @@ class HttpSender
     }
 }
 ```
+
+## Ensuring consistency in the Isolated Worker model
+
+If `SendsAtomicWithReceive` was previously enabled in the in-process model (note that it is not enabled by default), maintaining that consistency guarantee in the isolated worker model may be important. However, achieving this consistency in the isolated worker model requires more than just adding an assembly attribute. To ensure consistency, consider the following options:
+
+- **Implement the [Outbox pattern](/architecture/consistency.md#transactions-outbox-pattern)** to enable consistent database and message queue operations. (Recommended)
+- Ensure that all receiver message handlers are [idempotent](/architecture/consistency.md#idempotency).
```

Comment:
```suggestion
If `SendsAtomicWithReceive` was previously [enabled in the in-process model](/nservicebus/hosting/azure-functions-service-bus/in-process#message-consistency) (note that it is not enabled by default), maintaining that consistency guarantee in the isolated worker model is important. 

Lower [transaction modes](/transports/transactions.md#transactions) can result in the duplication of outgoing messages otherwise known as [ghost messages](/nservicebus/concepts/glossary.md#ghost-message). To ensure that [consistency](/architecture/consistency) is maintained make sure that all involved message handlers are [idempotent](https://docs.particular.net/architecture/consistency#idempotency). 
```

Reply:
I was on the fence if we should repeat the options mentioned in https://docs.particular.net/architecture/consistency#idempotency but thought that just linking to them was enough?

Reply:
After reviewing this again I think you're right.  At first, I thought we should still mention outbox, but even outbox requires receivers to be idempotent to deal with duplicated messages.